das-query-engine/tests/integration/handle/test_query_action.py

```diff
@@ -33,12 +35,45 @@ def valid_event(self, action_type):
                 },
             }
         }
+    
+    @pytest.fixture
+    def query_list(self, action_type):
```

Comment:
Success are good an all for testing, but testing the invalid paths are very important as well.
If so, you would've caught the `return True` inside the for.